js/scripts.js

```diff
@@ -0,0 +1,14 @@
+function toggleMenu() {
+  document.getElementById('toggle-menu').classList.toggle('hidden');
+  document.getElementById('toggle-menu').classList.toggle('toggle-menu');
+  document.getElementById('desktop-nav').classList.toggle('hidden');
+  document.getElementById('toggle-btn').classList.toggle('hidden');
+  document.getElementById('logo').classList.toggle('hidden');
+}
+
+function mobileMenuOption(event, id) {
+  event.preventDefault();
+  toggleMenu();
+  const linkTo = document.getElementById(id);
+  linkTo.scrollIntoView();
+}
```

Comment:
## [OPTIONAL]

It would be a better practice if you use arrow functions in JavaScript 😉, if you want to read more about them please check [this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).